recipe/meta.yaml

```
@@ -20,8 +20,14 @@ package:
   version: {{ version }}
 
 source:
+{% if rc != None %}
+  git_url: https://github.com/pytorch/pytorch.git
+  git_rev: v{{ version.replace("rc", "-rc") }}
+{% else %}
```

I'd prefer building from tags, not just https://github.com/pytorch/pytorch.git, e.g.

```suggestion
{% if rc != None %}
  url: https://github.com//pytorch/pytorch/archive/refs/tags/v{{ version.replace("rc", "-rc") }}.tar.gz
  sha256: <hash>
{% else %}
```
the downside however is that this wouldn't include submodules (due to git limitations) and pytorch uses [them](https://github.com/pytorch/pytorch/blob/v2.5.1-rc1/.gitmodules) _a lot_. So this is probably the better trade-off after all.

---

yes indeed. Are you ok with it as it is?

---

Yes, let's leave it as-is.